torch/_inductor/codegen/cpp.py

```diff
@@ -1873,6 +1874,11 @@ def __init__(
         self.tiling_idx = tiling_idx
         metrics.generated_cpp_vec_kernel_count += 1
 
+    def _reduction_var_name(self, var_name: str, line: str) -> str:
+        if f"{var_name}_vec" in line:
```

Comment:
Is this check mandatory?

Reply:
Yes, for `VecKernel`. There will be both `tmp0_acc` and `tmp0_acc_vec` in `stores`.